package/butterfly_angular/test/butterfly_angular_test.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2017, jonahwilliams. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+import 'package:butterfly_angular/butterfly_angular.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('A group of tests', () {
+    Awesome awesome;
```

Comment:
Does this compile? Where does the `Awesome` class come from?

Reply:
Removed for now